internal/kibana/client.go

```diff
@@ -80,19 +80,19 @@ func CertificateAuthority(certificateAuthority string) ClientOption {
 	}
 }
 
-func (c *Client) get(resourcePath string) (int, []byte, error) {
+func (c *Client) Get(resourcePath string) (int, []byte, error) {
 	return c.sendRequest(http.MethodGet, resourcePath, nil)
 }
 
-func (c *Client) post(resourcePath string, body []byte) (int, []byte, error) {
+func (c *Client) Post(resourcePath string, body []byte) (int, []byte, error) {
 	return c.sendRequest(http.MethodPost, resourcePath, body)
 }
 
-func (c *Client) put(resourcePath string, body []byte) (int, []byte, error) {
+func (c *Client) Put(resourcePath string, body []byte) (int, []byte, error) {
 	return c.sendRequest(http.MethodPut, resourcePath, body)
 }
 
-func (c *Client) delete(resourcePath string) (int, []byte, error) {
+func (c *Client) Delete(resourcePath string) (int, []byte, error) {
 	return c.sendRequest(http.MethodDelete, resourcePath, nil)
 }
```

Comment:
In order to be used in `internal/kibana/test/http_test.go` I had to set get method as public. The others I've changed it to ensure consistency.